intellij/src/saros/intellij/editor/annotations/AbstractEditorAnnotation.java

```diff
@@ -15,6 +15,7 @@
 import org.jetbrains.annotations.Nullable;
 import saros.filesystem.IFile;
 import saros.intellij.runtime.EDTExecutor;
+import saros.repackaged.picocontainer.Disposable;
```

Comment:
Is this really the Disposable interface you want to use ?

Reply:
I know what you mean. This isn't really the normal use case of the `Disposal` interface as it is never registered with a context.

I first wasn't sure whether the abstract parent class would have an implementation for the method, so I added the interface. It is probably better just to create a `protected` disposal method in the parent class and remove the interface as it is not really necessary.

Reply:
Done.